rest/endpoints/playback_info/playback_info.yml

```diff
@@ -113,12 +113,59 @@ paths:
 
     get:
       description: |
-        * **Available since:** NOT YET AVAILABLE
-      summary: # TODO
+        * **Available since:** 2020-01 (BPLAT-12230)
+      summary: Get the current playback info for one given asset
       tags:
         - Playback Info
       operationId: getPlaybackInfo
-      # TODO
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to get the playback info for.
+          schema:
+            type: string
+        - in: path
+          name: asset_type
+          required: true
+          description: The asset type of the playback info.
+          schema:
+            type: string
+            enum: ["recording", "catchup", "vod"]
+        - in: path
+          name: asset_id
+          required: true
+          description: The asset id of the playback info.
+          schema:
+            type: string
+
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    $ref: "../../snippets/PlaybackInfo/PlaybackInfoEntity.yml#/PlaybackInfoEntity"
```

Comment:
Not sure if this works ?

Reply:
Garrick used it in the other endpoint, so apparently it does